app/workers/data_migration_worker.rb

```diff
@@ -0,0 +1,23 @@
+class DataMigrationWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :high_priority, retry: 5
+
+  def perform
+    RailsDataMigrations::Migrator.list_pending_migrations.sort_by(&:version).each do |m|
```

Comment:
For reference for other reviewers:

https://github.com/OffgridElectric/rails-data-migrations/blob/4b893f9d2e5dd55e198f8edb9076697f004381ae/lib/tasks/data_migrations.rake#L4-L11